tests/util/test_network_protocol_files.py

```diff
@@ -24,468 +24,468 @@ def test_protocol_bytes() -> None:
         input_bytes = f.read()
 
     message_bytes, input_bytes = parse_blob(input_bytes)
-    message = type(new_signage_point).from_bytes(message_bytes)
-    assert message == new_signage_point
-    assert bytes(message) == bytes(new_signage_point)
+    message_0 = type(new_signage_point).from_bytes(message_bytes)
+    assert message_0 == new_signage_point
+    assert bytes(message_0) == bytes(new_signage_point)
```

Comment:
I _really_ wish I hadn't gotten into so many changes when I reworked this.  If I'd taken it piece by piece then we wouldn't even have this code...  `:[`